GW2EIParser/MainForm.Designer.cs

```diff
@@ -34,343 +34,356 @@ protected override void Dispose(bool disposing)
-        this.BtnCancelAll = new System.Windows.Forms.Button();
-        this.BtnSettings = new System.Windows.Forms.Button();
-        this.BtnClearAll = new System.Windows.Forms.Button();
-        this.DgvFiles = new System.Windows.Forms.DataGridView();
-        this.LocationDataGridViewTextBoxColumn = new System.Windows.Forms.DataGridViewTextBoxColumn();
-        this.StatusDataGridViewTextBoxColumn = new System.Windows.Forms.DataGridViewTextBoxColumn();
-        this.ParseButtonState = new System.Windows.Forms.DataGridViewButtonColumn();
-        this.ReParseButtonState = new DataGridViewDisableButtonColumn();
-        this.OperatorBindingSource = new System.Windows.Forms.BindingSource(this.components);
-        this.TlpMainWindow = new System.Windows.Forms.ToolTip(this.components);
-        this.BtnClearFailed = new System.Windows.Forms.Button();
-        this.BtnDiscordBatch = new System.Windows.Forms.Button();
-        this.LogFileWatcher = new System.IO.FileSystemWatcher();
-        this.LblVersion = new System.Windows.Forms.Label();
-        this.LblWatchingDir = new System.Windows.Forms.Label();
-        this.ChkApplicationTraces = new System.Windows.Forms.CheckBox();
-        this.ChkAutoDiscordBatch = new System.Windows.Forms.CheckBox();
-        ((System.ComponentModel.ISupportInitialize)(this.NumericCustomPopulateLimit)).BeginInit();
-        ((System.ComponentModel.ISupportInitialize)(this.DgvFiles)).BeginInit();
-        ((System.ComponentModel.ISupportInitialize)(this.OperatorBindingSource)).BeginInit();
-        ((System.ComponentModel.ISupportInitialize)(this.LogFileWatcher)).BeginInit();
-        this.SuspendLayout();
+        components = new System.ComponentModel.Container();
+        var resources = new System.ComponentModel.ComponentResourceManager(typeof(MainForm));
+        FileDialog = new OpenFileDialog();
+        LblHeader = new Label();
+        BtnParse = new Button();
+        BtnPopulate = new Button();
+        NumericCustomPopulateLimit = new NumericUpDown();
+        LblCustomPopulateLimit = new Label();
+        BtnCancelAll = new Button();
+        BtnSettings = new Button();
+        BtnClearAll = new Button();
+        DgvFiles = new DataGridView();
+        LocationDataGridViewTextBoxColumn = new DataGridViewTextBoxColumn();
+        StatusDataGridViewTextBoxColumn = new DataGridViewTextBoxColumn();
+        ParseButtonState = new DataGridViewButtonColumn();
+        ReParseButtonState = new DataGridViewDisableButtonColumn();
+        OperatorBindingSource = new BindingSource(components);
+        TlpMainWindow = new ToolTip(components);
+        BtnClearFailed = new Button();
+        BtnDiscordBatch = new Button();
+        LogFileWatcher = new FileSystemWatcher();
+        LblVersion = new Label();
+        LblWatchingDir = new Label();
+        ChkApplicationTraces = new CheckBox();
+        ChkAutoDiscordBatch = new CheckBox();
+        BtnCheckUpdates = new Button();
```

Comment:
The button should be disabled while the parser is running (QueueOrRunOperation and _RunNextOperation).